scheduler/networktopology/network_topology.go

```diff
@@ -321,6 +324,58 @@ func (nt *networkTopology) ProbedCount(hostID string) (uint64, error) {
 	return probedCount, nil
 }
 
+// Neighbours gets the specified number of neighbors for root node.
+func (nt *networkTopology) Neighbours(root *resource.Host, number int) ([]*resource.Host, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), contextTimeout)
+	defer cancel()
+
+	networkTopologyKeys, err := nt.cache.Scan(pkgredis.MakeNetworkTopologyKeyInScheduler(root.ID, "*"), number)
+	if err != nil {
+		return nil, err
+	}
+
+	// If we can not get a sufficient number of neighbors from the cache, access redis to get them.
+	if len(networkTopologyKeys) < number {
+		networkTopologyKeys, _, err = nt.rdb.Scan(ctx, 0, pkgredis.MakeNetworkTopologyKeyInScheduler(root.ID, "*"), math.MaxInt64).Result()
```

Comment:
`math.MaxInt64` => `const valuse`

Reply:
define const maxScanCount = math.MaxInt64, refer to https://sourcegraph.com/github.com/cockroachdb/cockroach@04dd7ea2c52c5ff378f6a22f58f59de1835b3feb/-/blob